backend/start_server.py

```
@@ -2,39 +2,44 @@
+
 
 def check_ollama():
-    """Check if Ollama is runnning and start if needed"""
+    """Check if Ollama is running and start if needed"""
 
     try:
         requests.get("http://localhost:11434/api/tags", timeout=5)
         print("✅ Ollama is running!")
         return True
-    except request.exceptions.ConnectionError:
+    except request.exceptions.RequestException:
```

Ruff (0.12.2)

Undefined name `request`

(F821)